app/views/groups/show.html.haml

```diff
@@ -32,13 +34,16 @@
             %tab{heading: t(:label_map),
                 active: "tabs.map.active",
                 select: "select(\'map\')"}
-              .map-container
-                %map{"ng-if" => "(isActive(\'/map\') && (mapShowed = true)) || mapShowed"}
-                  %ui-gmap-google-map{options: "map.additional_options", center: "map.center", zoom: "map.zoom", styles: "map.styles", draggable: "true"}
-                    %map-osm-tiles
-                    %map-search
-                    %ui-gmap-markers{models: "mapMarkers", fit: "true",
-                    coords: "'self'", icon: "'icon'", click: "'reveal'"}
+              .map-container{"ng-if" => "(isActive(\'/map\') && (mapShowed = true)) || mapShowed"}
+                - if ContentConfig.open_street_map_enabled
+                  = render partial: 'map/open_street_map'
+                - else
+                  %map
+                    %ui-gmap-google-map{options: "map.additional_options", center: "map.center", zoom: "map.zoom", styles: "map.styles", draggable: "true"}
```

Comment:
Note: we should refactor this repeated code block in `app/views/groups/show.html.haml` and `app/views/map/index.html.haml` and move it into a partial. It needs a slightly different `models` attribute in each one on the `ui-gmap-markers` element.

Reply:
Done here: #5494